app/Domain/Integrations/Integration.php

```diff
@@ -53,7 +54,8 @@ public function __construct(
         public readonly string $description,
         public readonly UuidInterface $subscriptionId,
         public readonly IntegrationStatus $status,
-        public readonly IntegrationPartnerStatus $partnerStatus
+        public readonly IntegrationPartnerStatus $partnerStatus,
+        public ?DateTimeInterface $reminderEmailSent = null,
```

Comment:
This should not be part of the default constructor because it has a `with` method.